docs/KernelFeatures/UsingSecureVault.md

```diff
@@ -106,3 +106,63 @@ All the Secret Repository implementations should derive from the Secret Reposito
             return new byte[0];
         }
     }
+```
+
+## Accessing Secure Vault in non OSGI environment.
+
+SecureVault functionality can access in non OSGI mode by following the below steps.
+
+1. Add secure vault dependency to the client project.
+
+        <dependency>
+            <groupId>org.wso2.carbon</groupId>
+            <artifactId>org.wso2.carbon.securevault</artifactId>
+            <version>${carbon.kernel.version}</version>
+        </dependency>
+
+2. Add following system properties to the client program.
```

Comment:
Yes. That is better. Anyway the standard files are only below three files
1. master key file path
2. secret properties file path 
3. secure vault yaml file path
Those files can be put as arguments. The other file (keystore file) depends on the default implementation. It can vary on other implementations. So How should input that file?